webproject/src/main/java/com/google/sps/servlets/MyOrderServlet.java

```diff
@@ -29,5 +31,18 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         String firebaseResponse = Firebase.sendGetRequest("https://summer20-sps-47.firebaseio.com/orders.json?orderBy=%22userEmailRoomId%22&equalTo=%22" + userEmail + "_" + roomId + "%22");
         response.setStatus(200);
         response.getWriter().print(firebaseResponse);
+    }     
+    
+    @Override
+    public void doDelete(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        BufferedReader reader = request.getReader();
+        String orderId = reader.readLine();
+        if (orderId == null) {
+            showMessageDialog(null, "orderID not found");        
+        }
+        orderId = orderId.split("=")[1];
```

Comment:
what happen if the data sent is:

{
  orderId: "../UserRoom/randomuserrromid"
}
?
I think we should check if the orderId is valid first before deleting it

Reply:
It wouldn't delete the room, since it would be retrieving from orders/UserRoom/randomuserroomid, but I see how that could be dangerous! I will change this

Reply:
I've added a check!